src/StoreitValue.js

```diff
@@ -51,12 +56,16 @@ export default class StoreitValue {
         this._store.set(this._key, update);
     }
 
+    toObject() {
+        return this.isStored ? this._getFromStore() : { [this._primaryKeyName]: this.key };
```

Comment:
Line is too long.<br>'computed property names' is only available in ES6 (use esnext option).